app/views/cases/index.html.erb

```diff
@@ -1,5 +1,9 @@
 <h1>Cases</h1>
-<table class="table table-striped">
+<%= render partial: "search_form" %>
+  <% if @cases.blank? %>
+    <%= flash[:notice] %>
+  <% else %>
```

Comment:
Create a partial for flash message display and use it in application.html.erb. Then it will be available in whole application